exchanges/kucoin/kucoin_wrapper.go

```diff
@@ -341,6 +341,40 @@ func (ku *Kucoin) UpdateTradablePairs(ctx context.Context, forceUpdate bool) err
 	return nil
 }
 
+// GetFuturesTickers returns the tickers for all currency pairs of the futures asset type
+func (ku *Kucoin) GetFuturesTickers(ctx context.Context) ([]*ticker.Price, error) {
+	var errs error
+	pairs, err := ku.GetEnabledPairs(asset.Futures)
+	if err != nil {
+		return nil, err
+	}
+	tickers := make([]*ticker.Price, 0, len(pairs))
+	for i := range pairs {
+		p, err := ku.FormatExchangeCurrency(pairs[i], asset.Futures)
+		if err != nil {
+			errs = common.AppendError(errs, err)
+			continue
+		}
+		fTick, err := ku.GetFuturesTicker(ctx, p.String())
+		if err != nil && !errors.Is(err, currency.ErrPairNotFound) {
```

Comment:
`!errors.Is(err, currency.ErrPairNotFound)` won't ever be utilised

Reply:
removed